app/controllers/statistics_controller.rb

```diff
@@ -12,6 +14,31 @@ def collection
     @all_indexers     = build_user_array(DeedType::PAGE_INDEXED)
   end
 
+  def export_csv
+    header = ['User Login', 'User Name', 'Email', 'Opt-In']
+    owner = User.find 'ushmmarchives'
```

Comment:
Get the user.  (This will need to be changed to the current user, or some parameterized thing.)

Reply:
@benwbrum done as well. :+1: